### ISSUE TITLE
Login Failed log?

### ISSUE THREAD

**Issue by phyyou** [simpleblog-project/Simple-Blog#1]
```javascript
$.ajax({
      method: "POST",
      url: "http://localhost:5000/auth/login",
      data: JSON.stringify({
        "name"     : id,
        "password" : password
      }),
      contentType: 'application/json'
    })
    .done(function(msg) {
      if (msg.access_token) {
        createCookie(msg.access_token);
        window.location.href = './Main.html';
      }
      else { 
        if(msg.message)
        {
          console.log(msg.message);
          alert(msg.message);
        }
      }
    });
```
this else{
if(msg.message)
{
  console.log(msg.message);
  alert(msg.message);
}
}
that is not working.
log was __jquery-3.3.1.min.js:2 POST http://localhost:5000/auth/login 400 (BAD REQUEST)__
this problem is related by app.py this part ▼
```python
@app.route('/auth/login', methods=['POST'])
def login():
    data = request.json
    name = data['name']
    password = data['password']
    user = User.query.filter_by(name=name).first()
    if user is None or not User.verify_password(user, password):
        return {"message": "invalid username or password"}, 400

    return {
        'access_token': create_access_token(user.id, expires_delta=access_token_exdelta),
        'refresh_token': create_refresh_token(user.id, expires_delta=refresh_otken_exdelta)
    }
```
so I create new branch "LoginFailBugFix"

if you wanna help this problem. checkout LoginFailBugFix

**Comment by hwc9169** [comment]
I love 박효신

**Comment by phyyou** [comment]
Work this : Flask doesn't return 200, if Success .then. else .catch callback.

**Comment by phyyou** [comment]
https://stackoverflow.com/questions/63594162/jquery-dont-send-msg-message 
"also might want to add a error part to the ajax call, just to check if the ajax call itself went ok."
```javascript
 $.ajax({
    method: "POST",
      url: "http://localhost:5000/auth/login",
      data: JSON.stringify({
        "name"     : id,
        "password" : password
      }),
      contentType: 'application/json',
    success: function (data, status, xhr) {
        console.log("Succes!" + data);
    },
    error: function (xhr) {
        console.log("Error happened" +xhr.status);

    }
});
```
Look at the documentation how the ajax call works: https://api.jquery.com/jQuery.ajax/

**Comment by phyyou** [comment]
It Fixed By #3 . merged.
